pkg/openstack/common.go

```diff
@@ -746,3 +782,15 @@ func GetIssuerCertSecret(
 	}
 	return issuer.Spec.CA.SecretName, nil
 }
+
+func hasCertInOverrideSpec(overrideSpec route.OverrideSpec) bool {
```

Comment:
should we have a kuttl test which provides certs via the overridespec?

Reply:
I did that in first commit, but the linter was failing for detecting raw key, so I just deleted it. If would just override the linter here it would fail in upcoming PRs, right?

Reply:
oh, you still have it somewhere so that we can check what the exact thing is it complains?

Reply:
I added new commit with updated kuttl test with tls data in overridespec, so let's see